lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -67,18 +68,27 @@ sub _init_db {
         my $dbclass = Zonemaster::Backend::DB->get_db_class( $dbtype );
         $self->{db} = $dbclass->from_config( $self->{config} );
     };
+
     if ($@) {
-        handle_exception('_init_db', "Failed to initialize the [$dbtype] database backend module: [$@]", '002');
+        handle_exception('_init_db', "Failed to initialize the [$dbtype] database backend module: [$@]");
     }
 }
 
 sub handle_exception {
-    my ( $method, $exception, $exception_id ) = @_;
+    my ( $_method, $exception ) = @_;
```

Comment:
Maybe you can remove the `$_method` since it is not used, my understanding is that `_build_method()` retrieve the method's name automatically.

Reply:
Done